integration-test-suite/cds-toolkit-integration-test/src/test/groovy/org/wso2/cds/integration/test/accounts/AccountsRetrievalRequestHeaderValidationTest.groovy

```diff
@@ -378,7 +378,7 @@ class AccountsRetrievalRequestHeaderValidationTest extends AUTest {
 
         def response = AURequestBuilder.buildBasicRequestWithCustomHeaders(userAccessToken,
                 AUConstants.X_V_HEADER_ACCOUNTS, clientHeader)
-                .queryParam(AUConstants.PARAM_PRODUCT_CATEGORY, "TRANS")
+                .queryParam(AUConstants.PARAM_PRODUCT_CATEGORY, "TANS")
```

Comment:
Is this a typo?

Reply:
No This is a test cases to validate the behaviour when we send an invalid product-category value.